ledger-core/rms/proto_vnvn.r0.proto

```diff
@@ -42,23 +55,23 @@ message VCallRequest {
     Reference ExpenseCenter = 31;
     Reference ResourceCenter = 32;
 
-    bytes PayloadHash = 33;
+    Binary PayloadHash = 33;
     CallDelegationToken DelegationSpec = 34;
-    bytes ProducerSignature = 36;
-    bytes RegistrarSignature = 37;
+    Binary ProducerSignature = 36;
+    Binary RegistrarSignature = 37;
     CallDelegationToken RegistrarDelegationSpec = 38;
 
     int32 CallRequestFlags = 40 [(gogoproto.casttype) = "CallRequestFlags"];
     Reference KnownCalleeIncoming = 41;
-    bytes EntryHeadHash = 42;
+    Binary EntryHeadHash = 42;
     Reference CallOutgoing = 43;
     uint32 TXExpiry = 44 [(gogoproto.casttype) = "PulseNumber"];
 
-    bytes SecurityContext = 45;
-    bytes TXContext = 46;
-    bytes Arguments = 47;
-    bytes ExtensionHashes = 48;
-    bytes Extensions = 49;
+    Binary SecurityContext = 45;
+    Binary TXContext = 46;
+    Binary Arguments = 47;
+    Binary ExtensionHashes = 48;
```

Comment:
please mark ExtensionHashes and Extensions as deprecated - they have to be removed

Reply:
it will be a part of RecordBody field (to be added)

Reply:
It's not used anywhere (except validations) so I can remove it without any problem